client/signup/steps/website-content/style.scss

```
@@ -0,0 +1,23 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.signup__step.is-site-info-collection {
```

```suggestion
.signup__step.is-website-content {
```

I think the step name was missed here.